Article/Article.js

```diff
@@ -3,18 +3,35 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+    
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelectorAll('.expandButton');
+
     // Using your expandButton reference, update the text on your expandButton to say "expand"
+
+    this.expandButton.forEach((button) => {
+      button.textContent = 'expand';
+    })
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
+    //expanding the articles upon clicking "expand"//
     
+    this.expandButton.forEach((button) => {
+      button.addEventListener('click', (event) => {
+        this.expandArticle(event);
+      })
+    })
   }
+  
 
-  expandArticle() {
+  expandArticle(){
     // Using our reference to the domElement, toggle a class to expand or hide the article.
+        //TOGGLE the articles//
 
+
+    console.log('toggle success');
```

Comment:
I really like that you are console logging here, some people miss out on this great step and then it can nip them later on